app/assets/javascripts/app/form-validation.js

```diff
@@ -2,7 +2,6 @@ import h5f from 'h5f';
 
 import validateField from './validate-field';
 
-
```

Comment:
we have a js linter but not for this particular thing -- it's just a style preference, i think all of our files with imports follow it